config-linux.md

```diff
@@ -158,13 +158,20 @@ cgroups provide controls to restrict cpu, memory, IO, pids and network for the c
 For more information, see the [kernel cgroups documentation][cgroup-v1].
 
 The path to the cgroups can be specified in the Spec via `cgroupsPath`.
-`cgroupsPath` is expected to be relative to the cgroups mount point.
-If `cgroupsPath` is not specified, implementations can define the default cgroup path.
+If `cgroupsPath` is
+* ... an absolute path (starting with `/`), the implementation MUST take the path to be relative to the cgroup mount point.
+* ... a relative path (not starting with `/`), implementations SHOULD take the value to be the suffix of the resulting cgroup path.
+* ... not specified, implementations CAN define the default cgroup path.
+Implementations CAN consider certain `cgroupsPath` values to be invalid, and MUST generate an error if this is the case.
+
 Implementations of the Spec can choose to name cgroups in any manner.
 The Spec does not include naming schema for cgroups.
 The Spec does not support [split hierarchy][cgroup-v2].
 The cgroups will be created if they don't exist.
 
+A runtime MUST at least use the minimum set of cgroups required to fulfil the resource restrictions specified by `config.json`.
+However, a runtime CAN decide to not attach to all cgroups supported by the system.
```

Comment:
On Wed, Jun 08, 2016 at 11:34:24AM -0700, W. Trevor King wrote:
> And while we're overhauling this language, we may want to address
> opting-out of runtime-managed cgroups (#237).

Actually, we may be able to avoid pulling that in here.  Your:

  * ... not specified, implementations CAN define the default cgroup path.

and:

  However, a runtime CAN decide to not attach to all cgroups supported
  by the system.

allow runtimes to handle unset cgroupsPath by joining whatever
controllers they want or none at all.  Since an explicit opt-out has
been contentious (although I still don't understand why), that
potential opt-out (at the runtime's discretion) is good enough for
now.